dandi/validate.py

```diff
@@ -22,10 +24,8 @@ def validate_bids(
         BIDS schema version to use, this setting will override the version specified in the dataset.
     devel_debug : bool, optional
         Whether to trigger debugging in the BIDS validator.
-    report_path : bool or str, optional
-        If `True` a log will be written using the standard output path of `.write_report()`.
-        If string, the string will be used as the output path.
-        If the variable evaluates as False, no log will be written.
+    report : bool or str, optional
```

Comment:
Why would it explicitly check if it's a string? Any true value is true.

Reply:
do you have a test where report is saved into specified here `str`?

I see that below if is assigned to `report_path = "{log_dir}/bids-validator-report_{{datetime}}-{{pid}}.log"` indeed for any `if report` thus if I say `report="here.log"` it would still save into that path, or am I wrong?

Reply:
the docstring is outdated.